bridge_adaptivity/module/static/module/js/module.js

```diff
@@ -131,7 +137,7 @@
                 $("<span/>")
                     .data("toggle", "tooltip")
                     .attr("title", item.type)
-                    .addClass("badge pull-left glyphicon " + typeToIcon[item.type])
+                    .addClass("badge pull-left glyphicon " + typeToIcon(item.type))
```

Comment:
Maybe template strings more suitable?

Reply:
Yep, template string would be better to use.